vaadin-grid-flow-integration-tests/src/main/java/com/vaadin/flow/component/grid/it/dataview/AbstractItemCountGridPage.java

```diff
@@ -218,7 +218,8 @@ protected void switchToDataProvider() {
     }
 
     protected void switchToDefinedSize() {
-        grid.getLazyDataView().setItemCountCallback(dataProvider::size);
+        grid.getLazyDataView()
+                .setItemCountCallback(query -> dataProvider.size(query));
```

Comment:
Use method reference instead, meaning revert unnecessary change

Reply:
Hmm, why did I do this...Reverted